streams/src/test/java/org/apache/kafka/streams/tests/StreamsEosTest.java

```diff
@@ -49,12 +48,9 @@ public static void main(final String[] args) throws IOException {
         }
 
         if ("process".equals(command) || "process-complex".equals(command)) {
-            if (!StreamsConfig.EXACTLY_ONCE.equals(processingGuarantee) &&
-                !StreamsConfig.EXACTLY_ONCE_BETA.equals(processingGuarantee) &&
-                !StreamsConfig.EXACTLY_ONCE_V2.equals(processingGuarantee)) {
+            if (!StreamsConfig.EXACTLY_ONCE_V2.equals(processingGuarantee)) {
 
-                System.err.println("processingGuarantee must be either " + StreamsConfig.EXACTLY_ONCE + " or " +
-                                       StreamsConfig.EXACTLY_ONCE_BETA + " or " + StreamsConfig.EXACTLY_ONCE_V2);
+                System.err.println("processingGuarantee must be " + StreamsConfig.EXACTLY_ONCE_V2);
```

Comment:
nit: `processingGuarantee` -> `processing.guarantee`

Reply:
Actually correct as-is, as we refer to the variable `processingGuarantee` here -- note: it's internal test code; not user facing.